### PR TITLE
Updated some descriptions for some channels

### PR DESCRIPTION
**Pull request by tash2020**
closes #ISSUE-NO

#### What does this PR do?
I'll try to updated some descriptions that seems to be outdated and in my opinion, need to be changed. 

#### Questions:
I have a question for the #organisation channel. It seems like the description unlikely describe the channel wisely. Personally, I get confused about it. Please consider to updated the descriptions.

### CODE REVIEW

File: discord-readme.md
Comment (nhcarrigan) [inline discord-readme.md:65]:
```suggestion
-> #first-timers: discussion about getting :green_square: and also the issues that are mentioned in the #good-first-issues

```

File: discord-readme.md
Comment (nhcarrigan) [inline discord-readme.md:49]:
```suggestion
-> #bot-chat: discuss bots, esp. regarding development of EddieBot. This is also a place to check your profile and ranks in the community using the bot, or use the other bot commands.

```

File: discord-readme.md
Comment (nhcarrigan) [inline discord-readme.md:53]:
```suggestion
-> #healthfitness: share your tips on health and fitness so we can be healthy together.
```
`healthfitness` isn't really a word. Maybe "so we can be healthy together"?

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
Thanks for the PR - a couple of quick notes.

**Review by schmelto — COMMENTED** [review]
with @nhcarrigan changes it looks good for me 🚀

File: discord-readme.md
Comment (tash2020) [reply discord-readme.md:53]: Yeah, I agree with you. I'm just trying to match with the name's channel. It seems odd if we use the word healthfitness.

**Comment by tash2020** [comment]
@nhcarrigan Thank you for your suggesstions. Those changes really sound great. 

So what should I do? should I click commit suggestion button?

**Comment by schmelto** [comment]
> And what I need to do to apply those changes? should I click commit suggestion button or anything else?

You can simply click on "commit suggestion" or you can add those manually. Suggestion is probably the easier way :)

**Comment by tash2020** [comment]
@schmelto I've done it 😄 Thanks

and what I need to do next? 

(This is my first PR sorry if I asked too much)

**Comment by schmelto** [comment]
> @schmelto I've done it 😄 Thanks
> 
> and what I need to do next?
> 
> (This is my first PR sorry if I asked too much)

No problem 👍🏼 now some community members will check your changes and approve them. If everything is ok. A member with write access will merge the PR and everything is done.

File: discord-readme.md
Comment (schmelto) [inline discord-readme.md:50]:
```suggestion
```

File: discord-readme.md
Comment (schmelto) [inline discord-readme.md:67]:
```suggestion
```

**Review by schmelto — APPROVED** [review]
Looking good for me 🚀. Maybe you can remove those empty lines (just a minor thing).

**Comment by tash2020** [comment]
@schmelto I get nervous easily because of that. I appreciated your value. Many thanks!

> Looking good for me 🚀. Maybe you can remove those empty lines (just a minor thing).

I will

**Comment by nhcarrigan** [comment]
@eddiejaoude I'm going to hold off on merging this as you're the only one with perms to edit the content in the Discord channel.

**Review by adityaraute — APPROVED** [review]

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM

**Review by muthuannamalai12 — APPROVED** [review]
LGTM

**Review by srinath1412001 — APPROVED** [review]

**Comment by nhcarrigan** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

**Comment by tash2020** [comment]
@kaiwalyakoparkar @muthuannamalai12 Thank you!

@nhcarrigan I'm glad to hear that 🎉 Totally enjoyed the process. Will try to contribute as much as I could!

Anyway, thanks for all of your support folks! I got a confidence to contribute more in an open source.

**Pull request merged**
